nav2_system_tests/README.md

```diff
@@ -0,0 +1,30 @@
+# Testing
```

Comment:
Typically, these titles have the same name as the directory that they are READMEs for. In this case, it should be "System Tests" or "Nav2 System Tests" (we should be consistent). Perhaps we should rename this directory to be "nav2_tests". Should also mention more strongly that unit tests don't go here.

Reply:
Good point, I'll make a few changes to make it more clear.